Generate_tile.scad

```diff
@@ -69,7 +72,7 @@ difference () {
         minkowski() {
             cube([margin_length_offset,margin_width_offset,margin_height_offset]);
             // rounded corners
-            cylinder(r=rounding_factor,h=rounding_factor);
```

Comment:
The rounding_factor was affecting the overall tile height because it was used as height